infinite-gas.md

```diff
@@ -0,0 +1,108 @@
+Infinite Gas
+Here we use the construct `#gas` to represent positive infinity, while tracking the gas formula through execution.
+This allows (i) computing final gas used, and (ii) never stopping because of out-of-gas.
+Note that the argument to `#gas(_)` is just metadata tracking the current gas usage, and is not meant to be compared to other values.
+such, any `#gas(G)` and `#gas(G')` are the _same_ positive infinite, regardless of the values `G` and `G'`.
+In particular, this means that `#gas(_) <Int #gas(_) => false`, and `#gas(_) <=Int #gas(_) => true`, regardless of the values contained in the `#gas(_)`.
```

Comment:
```suggestion
Infinite Gas
============

Here we use the construct `#gas` to represent positive infinity, while tracking the gas formula through execution.
This allows (i) computing final gas used, and (ii) never stopping because of out-of-gas.
Note that the argument to `#gas(_)` is just metadata tracking the current gas usage, and is not meant to be compared to other values.
such, any `#gas(G)` and `#gas(G')` are the _same_ positive infinite, regardless of the values `G` and `G'`.
In particular, this means that `#gas(_) <Int #gas(_) => false`, and `#gas(_) <=Int #gas(_) => true`, regardless of the values contained in the `#gas(_)`.
```